model/src/main/kotlin/EvaluatorRun.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model
+
+/**
+ * The summary of a single run of the evaluator.
+ */
+data class EvaluatorRun(
+        val errors: List<Error>
```

Comment:
The `Run` object should contain information about the environment like the others do, but we can add that later.
Consider renaming `errors` to `result` to align the naming.

Reply:
> The `Run` object should contain information about the environment like the others do, but we can add that later.

I was thinking of how the environment could influence the behavior of the evaluator, and I couldn't think of a realistic scenario, so I omitted the environment  for now.

> Consider renaming `errors` to `result` to align the naming.

Then the output reads a bit odd, as there is no indication that the serialized properties actually belong to an error object. And I wanted to avoid nesting errors into an otherwise empty result object for now.